third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -331,6 +331,25 @@ func resourceContainerCluster() *schema.Resource {
 				Default:  false,
 			},
 
+<% unless version == 'ga' -%>
+			"authenticator_groups_config": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
+				Default:  false,
```

Comment:
This is saying that the block defaults to the value `false` as if it was a boolean; the original PR was correct in omitting this field.
```suggestion
```

Reply:
So in case of default value for a block, it should be a structure? Just for my knowledge.

Reply:
Unfortunately, Terraform schema isn't able to express default values for blocks- it doesn't natively recognise nested blocks, so we need to pretend they're single-item lists.

Generally, we rely on the API interpreting an unsent block as taking the default value for that block.